### PR TITLE
add migration guide for graph.cool service

### PR DESCRIPTION
**Pull request by praveenweb**
### Description

This PR adds a migration guide to move away from graph.cool to Hasura. Covers data migration, Auth, Functions and Permissions.

Link to docs page - https://deploy-preview-4065--hasura-docs.netlify.app/graphql/manual/guides/integrations/graphcool-migration.html

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR.

### Affected components

- [x] Docs

### CODE REVIEW

File: docs/graphql/manual/guides/integrations/graphcool-migration.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+.. thumbnail:: ../../../../img/graphql/manual/guides/graphcool-export.png
+   :alt: Graph.cool Export
+
+Step 2: Start MySQL with Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+   docker run --name graphcool-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -p 3306:3306 -d mysql:latest --default-authentication-plugin=mysql_native_password
```
Comment (marionschleifer) [inline docs/graphql/manual/guides/integrations/graphcool-migration.rst:33]: Do we have to mention here about dependencies that need to be installed for this command?

File: docs/graphql/manual/guides/integrations/graphcool-migration.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+In case you have functions in Graph.cool, Hasura has an equivalent feature called :ref:`event triggers <event_triggers>`. Migrating this would involve taking your code and deploying it on a different platform (preferably serverless functions).
+
+Do note that for event triggers, the payload that Hasura sends might be different and you might have to change the way request body parameters are handled in your function code.
+
+
+Migrating auth
+--------------
+
+If you were using Auth0 with Graph.cool, the migration should be fairly straightforward. You can configure Hasura with Auth0 easily by following :ref:`this guide <guides_auth0_jwt>`.
```
Comment (marionschleifer) [inline docs/graphql/manual/guides/integrations/graphcool-migration.rst:107]: We should probably say something about other auth methods?

Comment (praveenweb) [reply docs/graphql/manual/guides/integrations/graphcool-migration.rst:113]: Added other auth method instructions

**Comment by praveenweb** [comment]
Closing this PR, since the guide has been published [here](https://dev.to/praveenweb/migrate-from-graph-cool-to-hasura-2mfi)

**Pull request closed without merging**